extra_data/tests/test_validation.py

```diff
@@ -26,6 +26,14 @@ def data_aggregator_file():
         yield path
 
 
+@fixture(scope='function')
+def mock_empty_file():
+    with TemporaryDirectory() as td:
+        path = osp.join(td, 'RAW-R0450-DA02-S00001.h5')
+        make_examples.make_sa3_da_file(path, ntrains=0)
+        yield path
```

Comment:
Duplicate of the fixture in `conftest.py`?

Reply:
Yep, I'll fix that.